ui.frontend/src/view/FormFieldBase.js

```diff
@@ -450,14 +450,15 @@ class FormFieldBase extends FormField {
      * @param {Object} state - The state object.
      */
     updateEnabled(enabled, state) {
+        const screenReaderText = window.DOMPurify ? window.DOMPurify.sanitize(state.screenReaderText, { ALLOWED_TAGS: [] }) : state.screenReaderText;
```

Comment:
Let's move this code out of FormFieldBase.js. You don't need to remove aria-label attribute here

Reply:
Done!